##MAE 133A S23 Design Project Group

Member names: Nathan Ge, Andrew Gautier

(Make a copy of this file and include your group number in the filename.)

##Library Imports

In [ ]:
!pip install CoolProp # use pip to install Coolprop onto Google's virtual machine
import CoolProp
import plotly.graph_objects as go
from CoolProp.CoolProp import PropsSI
import numpy as np
import scipy
from scipy import constants
from scipy import constants
# After running the code, hover over the left side to see the value
sigma = constants.value(u'Stefan-Boltzmann constant') #in W/(m^2 K^4)

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.1/6.1 MB 37.6 MB/s eta 0:00:00


##Project Description
**Overall Problem Statement**: The goal of this project is to develop a conceptual design of a solar methane reactor that makes both graphite and hydrogen. The objective is to produce at least 1 kg of hydrogen and 3 kg of graphite from 15 kW of solar light input in one solar 'day' (10 hours) (note: the ‘downstream’ purification process and uses are not part of the design).

We seek to accomplish this decarbonization using direct solar-thermal pyrolysis (see [this recent UCLA paper](https://pubs.acs.org/doi/abs/10.1021/acs.energyfuels.1c04405) as an example of such a process).

One set of tasks is to develop appropriately sized reactor, particularly its diameter and operating pressure (see the example sketches below). The input methane mass flow rate is known and fixed at 0.5 kg/hr. You have the design freedom to change the concentration optics within the range $C_s = 1000$ to 4000 suns for a fixed solar light input power of $Q_s = 15$ kW (note: 1 sun = 1 kW/m$^2$). A higher concentration requires a smaller reactor diameter, which in turn will increase the gas flow velocity, thus decreasing the residence time of the gases in the reaction zone. A low residence time can cause incomplete reaction, and our design goal is to maximize the daily hydrogen production rate. If the concentration is too low, then the reactor temperature may become too low for sufficient reaction.

The methane decomposition reaction is endothermic:
$$
\mathrm{CH_4 \rightarrow C + 2H_2} \quad (\bar{h}_i^o = 74.6 \hspace{4pt} \mathrm{kJ/mol})
$$
Your analysis will also include an assessment of exergy destruction.

**Main Assumptions** (also see the figures below):
* The reactor pressure is fixed at $p = 25$ torr because higher pressures tend to push the reaction backwards according to [Le Chatelier's principle](https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Equilibria/Le_Chateliers_Principle).   
* For a simple explainer of methane pyrolysis, see: https://www.advancedsciencenews.com/decarbonizing-natural-gas-methane-fuel-without-carbon-dioxide/.  
* The produced hydrogen can be treated as an ideal gas.
* The porous carbon felt absorber has a porosity of 0.9 and a thickness of 3.2 mm.

Laboratory system schematic:
![picture1](https://drive.google.com/uc?id=1jlbv8sS2zir6NQSL6Y563kcURiiYiQIa)

Thermal transport schematic:
![picture2](https://drive.google.com/uc?id=1jmVMzmSpz7VR5xd-TNe6YGfZ0i1abjSq)

**Potentially Useful Background Materials**:
* A popular science article on natural gas decarbonization: https://www.scientificamerican.com/article/can-natural-gas-be-part-of-a-low-carbon-future/.
* A paper on the exergy value of incident solar irradiation (available to all students when on a campus network or VPN): https://aip.scitation.org/doi/10.1063/1.3081637.
* A good compilation of various resources on incident solar energy by location: https://www.nrel.gov/gis/solar.html.

**Project Format**:
Projects must be submitted in the form of a Colab notebook. This is the template.  



##Questions



1. What is the required methane flow rate in the common units of 'standard cubic centimeters per minute' (sccm) used by mass flow controllers?  

In [ ]:
m_dot = 0.5 #kg/hr
ρ_CH4_stp = PropsSI('D', 'P', 101325, 'T', 273.15, 'CH4') #kg/m^3
v_CH4_stp = 1/ρ_CH4_stp

u_dot = m_dot / ρ_CH4_stp * 10**6 /60

print(ρ_CH4_stp)
print(u_dot)

0.7174587771429166
11615.069184209517


Answer 1: 11615.069184209517 sccm

2. Laboratory experiments have been conducted to form a relationship between simulated solar irradiation and the temperature of the felt absorber substrate. Each experiment was conducted at a pressure of 25 torr and a methane flow rate of 100 sccm, with the following results:

| Solar power, $Q_{s,exp}$ (kW) | Felt T (K) |
| --- | --- |
| **2.3193**	|	**1758**|
|1.5475	|	1575 |
| 1.8625	|	1658 |
| 2.1775	|	1735 |
| 2.2563	|	1747 |
| 1.8625	|	1658|

The effective light absorption area in these experiments was $A_{abs} = 10$ cm$^2$. Our task is to estimate the effective solar absorptivity $\alpha_s$ in:

$$
Q_{abs} = \alpha_s Q_{s,exp}
$$

The heat losses can be assumed to come from infrared radiation emission from both sides of the absorber, assumed to be a blackbody in the infrared band:

$$
Q_{loss} = 2 A_{abs} \sigma (T^4 - T_{\infty}^4)
$$
where $\sigma$ is the Stefan-Boltzmann constant:
$$
\sigma = 5.670374419\times 10^{-8} \mathrm{ W m^{-2} K^{-4}}
$$
(note: this constant has been included in the code above as 'sigma', drawn from the scipy constants library), and $T_{\infty}$ can be assumed to be room temperature (because the chamber is water-cooled).

Assuming complete reaction in these experiments, estimate a single value for solar absorptivity, $\alpha_s$ by applying a complete 1st law analysis of the process (including enthalpies of the gas reactant entering at room temperature and the gas product exiting at the absorber temperature).








In [ ]:
Q_s_exp = [2.3193, 1.5475, 1.8625, 2.1775, 2.2563, 1.8625]  #kW
T_felt = [1758, 1575, 1658, 1735, 1747, 1658] #K

A_abs = 10/10000 #m^2
σ = 5.670374419*10**-8 #Wm^−2K^−4
T_inf = 293.15 #K

m_dot = ρ_CH4_stp * 100 / 1e6 /60  #kg/s
n_dot_CH4 = m_dot/ 16.04 * 1000 #mol/s
n_dot_H2 = 2* n_dot_CH4
α_s = []
cp_carbon = 0.71 #kJ/(kgK)

h_f_CH4 = -74850
for i in range(len(Q_s_exp)):
  carbon_energy = n_dot_CH4*12*cp_carbon*(T_felt[i]-298)
  H_in = PropsSI('HMOLAR', 'P', 3333.06, 'T', 293.15, 'CH4') - PropsSI('HMOLAR', 'P', 101325, 'T', 298, 'CH4') + h_f_CH4  #J/mol
  H_out = PropsSI('HMOLAR', 'P', 3333.06, 'T', T_felt[i], 'H2') - PropsSI('HMOLAR', 'P', 101325, 'T', 298, 'H2') #J/mol
  Q_loss = 2*A_abs*σ*(T_felt[i]**4 - T_inf**4)  #W
  α_s = np.append(α_s, (Q_loss - (n_dot_CH4*(H_in) - n_dot_H2*(H_out) - carbon_energy))/ (Q_s_exp[i]) / 1000)

print(α_s)
α_s_avg = np.mean(α_s)
print(α_s_avg)


[0.47237514 0.45828361 0.46646858 0.47755044 0.47363381 0.46646858]
0.46913002845755


Answer 2: 0.469


3. Now, we need to develop an understanding of chemical kinetics. For the first case (**in bold**) in the table of Question 2, calculate the residence time of the methane stream in the porous absorber (i.e., the average amount of time that a methane molecule spends traversing through the felt). Assume that in this region the methane is at the same temperature as the absorber.


In [ ]:
Q_s_exp = 2.3193
T_felt = 1758
porosity = 0.9

ρ_CH4 = PropsSI('D', 'P', 3333.06, 'T', T_felt, 'CH4')
L = .0032 #m
v = m_dot/ (ρ_CH4 * A_abs)
τ_res = L / v
τ_res /= porosity

print(τ_res)

0.010877423907384722


Answer 3: 0.01088 seconds


4. More detailed measurements of the case in Question 3 reveal that the product stream actually contains molar concentrations of 98% hydrogen and 2% methane. We will assume that the reaction rate constant $k$ takes an Arrhenius form:
$$
k = C_0 \exp{\frac{-E_a}{R T}}
$$
where the activation energy is known to be $E_a = 250$ kJ/mol. We will take a simple approach for this ultra-fast reaction such that overall rate of reaction is described by a first-order reaction:
$$
\frac{d[CH4]}{dt} = k [CH4]
$$
where $[CH4]$ is the molar concentration (mol/L) of methane. Using the residence time from Question 3, calculate the prefactor $C_0$ in the rate constant.


In [ ]:
E_a = 250 #kJ/mol
R = 8.314
T = 1758

# integrate d[CH4]/dt = k[CH4] and solve for C_o.
# assume the process is adiabadic so we can use (0.02)/1 as an aproximation of the
# molarity after the reaction over the molarity before
C_o = np.log(.02/1) / (τ_res * np.exp(-E_a/(R*T)) )
print(C_o)

-365.8505807104166


Answer 4: -365.85058


5. We now have a reasonable engineering model of our reacting system with both thermal (Question 2) and chemical kinetics (Questions 3 and 4) factors determined. Using the methodologies above, **plot the absorber temperature as a function of solar concentration factor, $C_s$, between 1000 and 4000 suns under the conditions of this problem**. Remember, each different concentration factor will require a different reactor diameter because the total solar power is fixed, and each diameter will have a different residence time.



In [ ]:
# define constants that will be used in finding the state of the system
Q_solar_in = 15 #kw
Q_abs = α_s_avg * Q_solar_in #kW (approximated with the average absorbtivity)

L = .0032 #m
m_dot = 0.5 #kg/hr
m_dot = m_dot/60/60 #kg/s

σ = 5.670374419*10**-8 #Wm^−2K^−4
T_inf = 293.15 #K

M_CH4 = 16 #kg/kmol
n_dot_CH4_in = m_dot/M_CH4 #kmol/s
#print("n_dot_CH4_in", n_dot_CH4_in)

p_system = 25*133.322 #Pa 3331
T_ref = 298
p_ref = 101325

# find Absorber Temperature takes in a sunlight concentration factor
# a bool that represents if it should acount for chemical enthalpy
# and an initial temperature to calculate the temperture of the
# carbon felt iteratively
def findAbsorberTemperature(concentration_factor, account_for_chemical, Temp_guess):
  area_absorber = Q_solar_in/concentration_factor

  felt_change_in_energy = .0101 #want this to be close to 0 for steady state this [kW]
  T_felt = Temp_guess
  T_ref = 298 #used for calculating absolute enthalpy

  molar_concentration_CH4_out = 0

  #while loop iterates until the energy balance is within .01 kW
  while(abs(felt_change_in_energy) > .01):
    #calculate felt temperature based on calculated felt_change in energy
    T_felt += 50*felt_change_in_energy

    #determine the residence time of the methane
    density_CH4 = PropsSI('D', 'P', p_system, 'T', T_felt, 'CH4') #assume density is given at the temperature of the felt
    v = m_dot/ (density_CH4 * area_absorber) #m/s
    τ_res = L / v #s
    τ_res /= porosity
    #print("τ_res", τ_res)

    #determine the concentrations of hydrogen and methane leaving the reactor
    molarity_CH4_in = density_CH4/1000/16.04
    molarity_CH4_out = np.exp(C_o*np.exp(-E_a/(R*T_felt))*τ_res + np.log(molarity_CH4_in)) #found using equation for 4 with same assumption

    percent_CH4_converted = (molarity_CH4_in - molarity_CH4_out)/molarity_CH4_in

    n_dot_CH4_out = n_dot_CH4_in*percent_CH4_converted #kmol/s
    n_dot_CH4_converted = n_dot_CH4_in - n_dot_CH4_out
    n_dot_H2_out = 2*n_dot_CH4_converted           #kmol/s
    n_dot_C_out = n_dot_CH4_converted

    #determine the energy removed by the flow of gas
    h_f_CH4 = -74850 #kJ/kmol
    h_f_H2 = 0
    h_f_C = 0

    h_CH4_in = h_f_CH4 + PropsSI('HMOLAR', 'P', p_system, 'T', 293.15, 'CH4') - PropsSI('HMOLAR', 'P', p_ref, 'T', T_ref, 'CH4') #kJ/kmol
    h_CH4_out = h_f_CH4 + PropsSI('HMOLAR', 'P', p_system, 'T', T_felt, 'CH4') - PropsSI('HMOLAR', 'P', p_ref, 'T', T_ref, 'CH4') #kJ/kmol
    h_H2_out = h_f_H2 + PropsSI('HMOLAR', 'P', p_system, 'T', T_felt, 'H2') - PropsSI('HMOLAR', 'P', p_ref, 'T', T_ref, 'H2') #kJ/kmol
    carbon_energy = n_dot_CH4*12*cp_carbon*(T_felt-298)

    if (account_for_chemical):
      E_gasflow_carried_out = n_dot_H2_out*h_H2_out + n_dot_CH4_out*h_CH4_out + carbon_energy - n_dot_CH4_in*h_CH4_in #kW
    else:
      E_gasflow_carried_out = 0

    #determine amount of energy lost via radiation
    Q_radiation_out = 2*area_absorber*sigma*(T_felt**4 - T_inf**4) / 1000 #kW

    #determine the change in energy of the felt control volume
    felt_change_in_energy = Q_abs - Q_radiation_out - E_gasflow_carried_out

  # determine solar to chemical efficiency after state has been found
  h_i = 74.6 * 1000 #kJ/kmol
  η_sc = ((n_dot_CH4_in - n_dot_CH4_out) * h_i)/ Q_solar_in
  return [T_felt, percent_CH4_converted, Q_radiation_out, E_gasflow_carried_out, n_dot_H2_out, n_dot_CH4_in, n_dot_CH4_out, η_sc]

In [ ]:
# create numpy arrays for the different properties we want to observe
C_s = np.arange(1000,4000, 1) #kW/m^2
Temperatures = np.zeros(len(C_s))
percent_CH4_converted = np.zeros(len(C_s))
Rad_losses = np.zeros(len(C_s))
gasflow_losses = np.zeros(len(C_s))
n_dot_H2 = np.zeros(len(C_s))
n_dot_CH4_ins = np.zeros(len(C_s))
n_dot_CH4_outs = np.zeros(len(C_s))
Temperatures_rad = np.zeros(len(C_s))
Solar_Chem_efficiency = np.zeros(len(C_s))

#initial guesses for the reactor
Temp_guess_rad_and_chem = 1200
Temp_guess_rad = 1200

#for all the different concentration factors find th state and save them to the arrays
for i in range(len(C_s)):
  results_rad_and_chem = findAbsorberTemperature(C_s[i], True, Temp_guess_rad_and_chem)
  results_rad = findAbsorberTemperature(C_s[i], False, Temp_guess_rad)
  Temp_guess_rad_and_chem = results_rad_and_chem[0]
  Temp_guess_rad = results_rad[0]
  Temperatures[i] = results_rad_and_chem[0]
  percent_CH4_converted[i] = results_rad_and_chem[1]
  Rad_losses[i] = results_rad_and_chem[2]
  gasflow_losses[i] = results_rad_and_chem[3]
  n_dot_H2[i] = results_rad_and_chem[4]
  n_dot_CH4_ins[i] = results_rad_and_chem[5]
  n_dot_CH4_outs[i] = results_rad_and_chem[6]
  Temperatures_rad[i] = results_rad[0]
  Solar_Chem_efficiency[i] = results_rad_and_chem[7]


# plot a couple relevant signals
# you can isolate signals on the plot by clicking them on the legend
fig = go.Figure()
fig.add_trace(go.Scatter(x=C_s, y=Temperatures, mode='lines', name='Temperature from Radiation and Enthalpy'))
fig.add_trace(go.Scatter(x=C_s, y=Temperatures_rad, mode='lines', name='Temperature from Radiation only'))
fig.add_trace(go.Scatter(x=C_s, y=percent_CH4_converted, mode='lines', name='percent_CH4_not_converted'))
fig.add_trace(go.Scatter(x=C_s, y=Solar_Chem_efficiency, mode='lines', name='Solar_Chem_efficiency'))

fig.update_layout(
    title='Concentration Factor vs Temperature of Felt',
    xaxis_title='Concentration Factor',
    yaxis_title='Temperature [K]',
)

fig.show()

Answer 5:

6. In general, which energy loss term is larger, the radiative heat loss or the enthalpy difference between products (outlet) and reactants (inlet)? Explain your answer.

In [ ]:
fig = go.Figure()
fig.add_trace(go.Scatter(x=C_s, y=Rad_losses, mode='lines', name='Rad_losses'))
fig.add_trace(go.Scatter(x=C_s, y=gasflow_losses, mode='lines', name='gasflow_losses'))

fig.update_layout(
    title='energy losses for different concentration factors',
    xaxis_title='Concentration Factor',
    yaxis_title='Energy [kW]',
)
fig.show()

Answer 6: As shown from the graph the radiative losses are much larger than the losses from the change in enthalpy of the gas. However, losses from change in enthalpy of gas becomes more significant and radiative losses become less significant as concentration factor increases.

7. Based on your answers above, choose a concentration factor that will satisfy the requirement of producing 1 kg of hydrogen in a 10-hour period (the more the better). Explain why you chose this factor and include the resulting reactor diameter, temperature, and daily hydrogen production rate.  

In [ ]:
m_H2 = 1 #kg
t = 10 * 60 * 60 #s

m_dot_H2 = m_H2/t #kg/s
n_dot_H2_target = m_dot_H2/2.016 #kmol/s
print("Target H2 Molar Flow Rate:",n_dot_H2_target, "kmol/s")

for i in range(len(C_s)):
  if(np.abs(n_dot_H2[i] - n_dot_H2_target) <= 0.000000001):
    n_dot_H2_design = n_dot_H2[i]
    A_felt = Q_solar_in/C_s[i]
    reactor_Dia = np.sqrt(A_felt/(3.14159265))*2
    target_T = Temperatures[i]
    target_C_s = C_s[i]
    Radiative_loss = Rad_losses[i]
    percent_CH4_leftover = percent_CH4_converted[i]

    daily_hydrogen_production_rate = n_dot_H2_design*2.016*10*60*60
    print("Our Molar Flow Rate:", n_dot_H2_design, "kmol/s")
    print("percent CH4 leftover:", percent_CH4_leftover)
    print("Solar Concentration:", target_C_s)
    #print("Index: ", i)
    print("\n")
    print("Reactor Diameter:",reactor_Dia, " m")
    print("Felt Temperature:",target_T, " K")
    print("Daily Hydrogen Production Rate:", daily_hydrogen_production_rate, "kg")
    break


fig = go.Figure()
fig.add_trace(go.Scatter(x=C_s, y=n_dot_H2, mode='lines', name='n_dot_H2'))
fig.add_hline(y=n_dot_H2_target)

fig.update_layout(
    title='Molar Flow Rate of H2 vs Solar Concentration',
    xaxis_title='Concentration Factor',
    yaxis_title='Molar Flow Rate [kmol/s]',
)







Target H2 Molar Flow Rate: 1.3778659611992946e-05 kmol/s
Our Molar Flow Rate: 1.3778877449407996e-05 kmol/s
percent CH4 leftover: 0.20633665891409947
Solar Concentration: 2366


Reactor Diameter: 0.08984487710041114  m
Felt Temperature: 1622.028753314315  K
Daily Hydrogen Production Rate: 1.0000158097682348 kg


Answer 7:
Concentration Factor: 2366

daily hydrogen production rate: 1 kg

reactor diameter: 0.09 m

We chose the lowest concentration factor because we think using a low concentration factor means its easier to construct the solar reactor while still getting the desired production rate. At this concentration factor 20% of the methane is unconverted but at 4000 suns 10% is converted which isn't that much of an improvement. It would probably be easier to get a higher production rate by building two reactors than increasing the solar concentration factor.

8. Using your design from Question 7, calculate the rate of exergy destruction in your process, including the exergy of the sunlight, the chemical exergies of the inlet methane and outlet gas mixture (assuming that it cools to room temperature), and radiative losses.

In [ ]:
#SUNLIGHT EXERGY
T_ref = 298 #K
T_sun = 6000 #K
I_0 = 1373 #W/m^2
I_T0 = 6.7*1000/24 #W/m^2 obtained from NREL direct solar irradiance for LA

#(C Zamfirescu)
η =  (1-T_ref/T_sun)*(I_0/I_T0)
exergy_sunlight = η*Q_solar_in #kW

#RADIATIVE LOSSES
T_felt = target_T
exergy_radiative = (1-T_ref/T_felt)*Radiative_loss

#CHEMICAL EXERGY
# Model IIb
R = 8.314
T_0 = 273.15 #K

y_H2 = 2/3
y_C = 1/3
e_ch_CH4 = 831650 + R*T_0*np.log(1/(y_H2**2 * y_C**1)) #kJ/kmol
e_ch_H2 = 236100 + R*T_0*np.log(1/y_H2**2) #kJ/kmol
e_ch_C = 410260 + R*T_0*np.log(1/y_C**1) #kJ/kmol

g_CH4 = -50790 #kJ/kmol
g_C = 0
g_H2 = 0
e_F = (g_CH4 - g_C - 2*g_H2) + e_ch_C + 2*e_ch_H2

#TOTAL FLOW EXERGY
h_CH4_in = PropsSI('HMOLAR', 'P', p_system, 'T', 293.15, 'CH4') - PropsSI('HMOLAR', 'P', p_ref, 'T', T_ref, 'CH4')
h_CH4_out = PropsSI('HMOLAR', 'P', p_system, 'T', T_ref, 'CH4') - PropsSI('HMOLAR', 'P', p_ref, 'T', T_ref, 'CH4')
h_H2 = PropsSI('HMOLAR', 'P', p_system, 'T', T_ref, 'H2') - PropsSI('HMOLAR', 'P', p_ref, 'T', T_ref, 'H2')

s_CH4_in = PropsSI('SMOLAR', 'P', p_system, 'T', 293.15, 'CH4') - PropsSI('SMOLAR', 'P', p_ref, 'T', T_ref, 'CH4')
s_CH4_out = PropsSI('SMOLAR', 'P', p_system, 'T', T_ref, 'CH4') - PropsSI('SMOLAR', 'P', p_ref, 'T', T_ref, 'CH4')
s_H2 = PropsSI('SMOLAR', 'P', p_system, 'T', T_ref, 'H2') - PropsSI('SMOLAR', 'P', p_ref, 'T', T_ref, 'H2')

e_CH4_in = e_ch_CH4 + h_CH4_in - 293.15*s_CH4_in
e_CH4_out = e_ch_CH4 + h_CH4_out - T_felt*s_CH4_out
e_H2 = e_ch_H2 + h_H2 - T_felt*s_H2
e_C = cp_carbon*(T_felt-T_0) - T_0*cp_carbon*np.log(T_felt/T_0) + e_ch_C

n_dot_CH4_out = n_dot_CH4_in - n_dot_H2_design/2
n_dot_C = n_dot_H2_design/2
exergy_destruction = exergy_sunlight - exergy_radiative + n_dot_CH4_in*e_CH4_in - n_dot_CH4_out*e_CH4_out - n_dot_H2_design* e_H2 - n_dot_C*e_C

print("exergy_destruction: ", exergy_destruction)



exergy_destruction:  66.32885562273128


Answer 8: 66.32886 kW

Sunlight exergy equation:
*   Zamfirescu, C., & Dincer, I. (2009). How much exergy one can obtain from incident solar radiation?. Journal of Applied Physics, 105(4), 044911.

Direct Normal Solar Irradiance:
*  Sengupta, M., Y. Xie, A. Lopez, A. Habte, G. Maclaurin, and J. Shelby. 2018. "The National Solar Radiation Data Base (NSRDB)." Renewable and Sustainable Energy Reviews  89 (June): 51-60.





9. For your process, calculate the solar-to-chemical efficiency, defined as:

1.   List item
2.   List item




$$
\eta_{sc} \equiv \frac{(\dot{n}_{CH4,in}-\dot{n}_{CH4,out}) \bar{h}_i^o}{\dot{Q}_{s}}
$$

In [ ]:
h_i = 74.6 * 1000 #kJ/kmol
η_sc = ((n_dot_CH4_in - n_dot_CH4_out) * h_i)/ Q_solar_in
print(η_sc)

0.034265853176937686


Answer 9: 3.427%

10. What are the technical and/or societal benefits and drawbacks of using solar-thermal energy in this design, as compared to burning methane or hydrogen for pyrolysis?

The largest benefit of this solar reactor is that it does not produce greenhouse gas emissions which are harmful to the planet. This makes the solar powered reactor the more sustainable long term choice of the two. Burning hydrogen created by the reactor is clean but it does mean we are getting rid of the product that we are trying to create. If the US were to transition to using more hydrogen fuel we would not want its price tied to the availability of hydrocarbons. While the reactor does use methane to create hydrogen, it would depend much less on its availability than its combustion counterpart, leading to less hydrogen gas price swings and what economic implications that may bring.

However, solar-thermal technology comes with its own set of drawbacks. This design is more technically complicated and novel than a regular combustion reactor. Combustion reactors can be built in most industrial areas or anywhere with access to natural gas. This is not the case for solar energy reactors.  One blatant limitation is that solar energy extraction is closely tied to weather conditions and day-night cycles, resulting in intermittent power generation. Not only does this necessitate energy storage to ensure a consistent energy supply (adding complexity and cost to the overall system), geographical location must be considered when constructing solar-thermal energy systems to optimize energy availibility. Location must also be decided on the basis of land availibility, as solar-thermal power plants require significant space for the installation of solar collectors or mirrors. Solar energy reactors have not been used industrially either while combustion reactors already exist. This means that there would be a higher upfront cost in developing the technology necessary to make this process work on an industrial scale.

Nevertheless, exploring this research is worth it as we try to transition to a society that is less dependent on fossil fuels. While it is easier to build combustion powered pyrolysis reactors in the present, that might not be true in the future. The deployment of solar-thermal technologies also paves the way for employment opportunities, decentralized power generation and subsequently local resilience, and energy diversification to bolster energy security. In addition, the technology developed in researching this process could be used for other applications.


11. More broadly, what are the technical and societal benefits and drawbacks of using methane (from natural gas or biogas) as a source for graphite and hydrogen?

Electrolysis uses electricity to break the bonds of water molecules and it is the only other industrially viable hydrogen production method. Methane and natural gas are widely available and cheap in comparison to electricity. This is why we use gas for stoves and heating, despite its health and climate impacts. While the actual process of electrolysis is clean, the electricity needed for it is not. Most of Americas electricity comes from burning fossil fuels so electrolysis is not cleaner than methane especially if the carbon from methane is deposited as graphite. Using biogas as a hydrogen production source would benefit the US economy since it is easily produced domestically, as the infrastructure to extract and harness methane is highly optimized. Using methane to produce hydrogen also means that graphite can be produced which is useful in many applications.

Furthermore, methane has a high gravimetric (energy content per unit mass) and volumetric (energy content per volume) energy density, making it an efficient fuel for various applications. This energy density allows for compact storage and transportation, making methane convenient for industrial use.

The biggest downside of methane usage is that it's inexplicably linked to global warming potential in the form of realsing CO2, NOx, and Methane itself into the atmosphere. Methane usage does not harness the possibility of cheap renewable energy. Right now most hydrogen is produced using steam methane pyrolysis. This process does not capture the carbon from methane and instead converts it to CO2 which is detrimental for the environment. As renewable energy becomes more widely available and cheaper, electrolysis will become cleaner than steam methane pyrolysis. Electricity will also become cheaper in the future which would make electrolysis more competetive with methane as a source of hydrogen.

## Appendix: Example Calculations
In the following, we analyze the process of methanol-to-hydrogen pyrolysis in order to find the heat transfer associated with the process using various assumptions that MIGHT NOT BE APPROPRIATE FOR THE PROBLEM. They are provided merely for reference in formulating your group's answers.   



### Problem Statement: Given
We consider a steady flow of room-temperature methane at pressure $P$ and flow rate $\dot{V}_{sccm}$, which will be expressed in units of `standard cubic centimeters per minute' (sccm). We assume (at least initially) that the methane is completely converted to hydrogen gas that discharges at steady flow as a product, and that the carbon is converted completely to solid form and collected by gravitational deposition beneath the reaction site.

Later, we could consider partial conversion of methane and/or the use of a moving web to remove the deposited carbon.

Some useful information follows:

*Molecular weights*

In [ ]:
M_CH4 = 16.043 #kg/kmol
M_H2 = 2.016
M_C = M_CH4 - 2*M_H2
2.*M_H2/M_CH4

0.2513245652309419

*Ideal gas constants*

In [ ]:
R = constants.value(u'molar gas constant') #kJ/(kmol K)
R_CH4 = R/M_CH4 #kJ/(kg K)
R_H2 = R/M_H2 #kJ/(kg K)
101325/(R_CH4*1000*298)

0.6560723679294734

*Density of methane as standard temperature and pressure (from CoolProp)*

In [ ]:
rho_CH4_STP = PropsSI('D','P',101325,'T',298,'methane') #kg/m^3
rho_CH4_STP

0.6572145035074058

In [ ]:
D_final = 15.488
P_final = PropsSI('P','D',D_final,'T',298,'hydrogen')
P_final

21572991.419918276

###Problem Statement: Find
We seek to find the net rate of heat transfer ($\dot{Q}_{net}$) required to sustain the reaction.

###Sketch

![picture2](https://drive.google.com/uc?id=1FI0yRsFX1KS2zmEU6CULFJTzoQ3n3NNC)


In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


###Engineering Model: Assumptions and Governing Equations
We will use real gas analysis (using CoolProp) under steady-flow conditions with methane entering at room temperature ($T_{CH4}$ = 298 K) and at a pressure that is to be specified in the analysis below. We will assume that changes in kinetic and potential energy are negligible, that no boundary work interactions occur, and that flow pressure drop is negligible. Finally, we will assume that the hydrogen leaves the reaction zone at an assumed pyrolysis temperature of $T_{H2}$ = 1200 C (= 1473 K) (note: the actual temperature above might be different).  

*Mass conservation*

We start with mass analysis:
$$
\frac{dm_{C}}{dt} = \dot{m}_{CH4} - \dot{m}_{H2}
$$
where we have assumed that the solid carbon ($m_C$) does not flow out of the control volume. As such, we need to match molar flow rates of the gas reactant and product.
$$
\dot{n}_{CH4} = 0.5\dot{n}_{H2}
$$
The steady energy flow balance is:
$$
\frac{dU_{C}}{dt} = \dot{Q}_{net} + \dot{m}_{CH4}h_{CH4} - \dot{m}_{H2}h_{H2}
$$
As a first approximation, we will assume that the internal energy of the solid carbon is negligible, such that the left side of the foregoing equation is zero. Also, we will need to calculate enthalpies using the enthalpy of formation ($\bar{h}_i^o$) to account for the chemical reaction:
$$
h_{i,eff}(T, P) = \bar{h}_i^o/M_i + h_i(T, P) - h_i(T_{ref}, P_{ref})
$$


###Analysis
The hydrogen flow rate can be expressed as:
$$
\dot{n}_{H2} = 2 \dot{n}_{CH4} = 2 \frac{\dot{m}_{CH4}}{M_{CH4}} = \frac{\dot{m}_{H2}}{M_{H2}}
$$
or,
$$
\dot{m}_{H2}= 2 \frac{M_{H2}}{M_{CH4}}\dot{m}_{CH4}
= 0.251 \dot{m}_{CH4}
$$

Again, neglecting the energy content of the solid carbon, the heat flow rate becomes:
$$
\dot{Q}_{net} = \dot{m}_{H2}h_{H2} - \dot{m}_{CH4}h_{CH4} = \dot{m}_{CH4}(0.251 h_{H2} - h_{CH4})
$$

*Example calculation*

We will start with a mass flow rate of $\dot{m}_{CH4} = 10^{-5}$ kg/s and an operating pressure of $p =$ 10,000 Pa.


In [ ]:
hform_CH4 = -74850000 / M_CH4 #h of formation number (-748...) is in units of J/kmol from Table A-25 of Moran and Shapiro
h_CH4 = hform_CH4 + PropsSI('H','P',10000,'T',298,'methane') - PropsSI('H','P',100000,'T',298,'methane')
print('h(CH4)', h_CH4, 'J/kg')
hform_H2 = 0.0
h_H2 = hform_H2 + PropsSI('H','P',10000,'T',1473,'hydrogen') - PropsSI('H','P',100000,'T',298,'hydrogen')
print('h(H2) = ', h_H2, 'J/kg')
Q_in = 1e-5 * (0.251*h_H2-h_CH4)
print('Q(into system)', Q_in, 'Watts')

h(CH4) -4664706.815820857 J/kg
h(H2) =  17576092.685188696 J/kg
Q(into system) 90.76306079803221 Watts


In [ ]:
PropsSI('D','P',3333,'T',298,'methane')

0.02158179985411088